tests/testsuite.src/run_file.at

```diff
@@ -13481,6 +13481,112 @@ AT_CHECK([$DIFF reference prog.out], [0], [], [])
 AT_CLEANUP
 
 
+# When the compiler is configured without a database package,
+# but an external file handler is specified at compile time,
+# programs with INDEXED files should NOT fail with the message
+# "runtime is not configured to support ORGANIZATION INDEXED"
+
+AT_SETUP([INDEXED file using EXTFH without database])
+AT_KEYWORDS([runfile EXTFH indexed without-db])
+
+# This test is only applicable if the compiler has been built
+# with the --without-db configuration option
+AT_SKIP_IF([test "$COB_HAS_ISAM" != "no"])
```

Comment:
we can always run this test as it should work in any case (and possibly extends EXTFH testing a bit as well)